### ISSUE TITLE
[Feature request] Request chaining (not just variable chaining)

### ISSUE THREAD

**Issue by dpaulino** [jenius-apps/nightingale-rest-api-client#145]
Basically, trigger a request before or after a particular request is sent.

**Comment by dpaulino** [comment]
Related to a comment in #144

**Comment by larsschellhas** [comment]
Ahhh... This explains why my later requests having difficulties with false tokens :D 

It would be nice if the requests were made one after another when I put them into a collection. Alternatively, that could be an option in or an extra button next to the "Execute All" button.
![image](https://user-images.githubusercontent.com/14281234/105709363-eb34f380-5f15-11eb-9652-4008a14c0fcd.png)

**Comment by dpaulino** [comment]
@larsschellhas 

> It would be nice if the requests were made one after another when I put them into a collection

Can you clarify what you mean by this? Currently, nightingale should be executing them one after another if you do click "execute all" from the collection page. Are you saying it does *not* execute one after the other for you?

**Comment by larsschellhas** [comment]
Hey @dpaulino,
sorry, if my description was not clear enough! Let me try again :)
I figured out what the real problem is and will create a new issue for it.

**Comment by fefemf10** [comment]
I have the same problem
I send a POST request from which I update the id variable
Then I want to send a deletion request by this id, if I do it via execute all, then the deletion request does not work
And if you manually take turns, then everything is fine
